_clients/agents-and-ingestion-tools/index.md

```diff
@@ -27,14 +27,12 @@ PUT _cluster/settings
 }
 ```
 
-For a longer term solution, we plan to create an OpenSearch output plugin for Logstash. This plugin *does not exist yet*, but we've included it in the compatibility matrices below based on its expected behavior.
-
-
 ## Downloads
 
-These versions of Logstash and Beats offer the best compatibility with OpenSearch. For more information, see the [compatibility matrices](#compatibility-matrices).
+You can download the OpenSearch output plugin for Logstash from [OpenSearch downloads](https://opensearch.org/downloads.html). The Logstash output plugin is compatible with OpenSearch and open source versions of Elasticsearch (7.10.2 or lower).
```

Comment:
My preference is "Elasticsearch OSS" just because it's concise and in line with the name of the artifact, rather than "open source versions of Elasticsearch."

Reply:
Changed.